inception-project/inception
# #4704 - Per-document agreement across all annotators

**What's in the PR**
- Fix warnings and count of unused annotations
- Improve styling

**How to test manually**
* Run agreement report using a coding measure and check report for consistency and interpretability

**Automatic testing**

**Documentation**
